doc/getting-started/roscube/roscube-gsg.rst

```diff
@@ -0,0 +1,720 @@
+   accelerate the installation time.  The settings here are not related to
+   the resource of the User VM on ACRN, which can be decided later.
+
+   .. figure:: images/rqi-acrn-kvm-cpu-ram.png
+
+#. Select disk size you want. **Note that this can't be modified after creating image!**
+
+   .. figure:: images/rqi-acrn-kvm-storage.png
+
+#. Edit image name and select "Customize configuration before install".
+
+   .. figure:: images/rqi-acrn-kvm-name.png
+
+#. Select correct Firmware, apply it, and Begin Installation.
+
+   .. figure:: images/rqi-acrn-kvm-firmware.png
+
+#. Now you'll see the installation page of Ubuntu.
+   After installing Ubuntu, you can also install some necessary
+   packages, such as ssh, vim, and ROS 2.
+   We'll clone the image for real-time VM to save time.
+
+#. To install ROS 2, please refer to `Installing ROS 2 via Debian Packages
+   <https://index.ros.org/doc/ros2/Installation/Dashing/Linux-Install-Debians/>`_
+
+#. Optional: Use ACRN kernel if you want to passthrough GPIO to User VM.
+
+   .. code-block:: bash
+
+     sudo apt install git build-essential bison flex libelf-dev libssl-dev liblz4-tool
+
+     # Clone code
+     git clone -b release_2.1 https://github.com/projectacrn/acrn-kernel
+     cd acrn-kernel
+
+     # Set up kernel config
+     cp kernel_config_uos .config
+     make olddefconfig
+     export ACRN_KERNEL_UOS=`make kernelversion`
+     export UOS="ROS2SystemUOS"
+     export BOOT_DEFAULT="${ACRN_KERNEL_UOS}-${UOS}"
+     sed -ri "/CONFIG_LOCALVERSION=/s/=.+/=\"-${UOS}\"/g" .config
+
+     # Build and install kernel and modules
+     make all
+     sudo make modules_install
+     sudo make install
+
+     # Update Grub
+     sudo sed -ri \
+       "/GRUB_DEFAULT/s/=.+/=\"Advanced options for Ubuntu>Ubuntu, with Linux ${BOOT_DEFAULT}\"/g" \
+       /etc/default/grub
+     sudo update-grub
+
+#. Poweroff the VM after complete.
+
+   .. code-block:: bash
+
+     sudo poweroff
+
+Run User VM
+===========
+
+Now back to the native machine to set up the environment for launching
+the User VM.
+
+#. Install dependency.
+   The origin version of iasl is too old for ACRN and should be upgraded.
+
+   .. code-block:: bash
+
+     sudo apt install iasl
+     cd /tmp
+     wget https://acpica.org/sites/acpica/files/acpica-unix-20191018.tar.gz
+     tar zxvf acpica-unix-20191018.tar.gz
+     cd acpica-unix-20191018
+     make clean && make iasl
+     sudo cp ./generate/unix/bin/iasl /usr/sbin/
+
+#. Convert KVM image file format.
+
+   .. code-block:: bash
+
+     mkdir -p ~/acrn/uosVM
+     cd ~/acrn/uosVM
+     sudo qemu-img convert -f qcow2 -O raw /var/lib/libvirt/images/ROS2SystemUOS.qcow2 ./ROS2SystemUOS.img
+
+#. Prepare a Launch Script File.
+
+   .. code-block:: bash
+
+     wget https://raw.githubusercontent.com/Adlink-ROS/ROScube_ACRN_guide/master/scripts/launch_ubuntu_uos.sh
+     chmod +x ./launch_ubuntu_uos.sh
+
+#. Set up network and reboot to take effect.
```

Comment:
@evshary , can you explain if this is any different? I'm thinking it may be beneficial to stick to the same method we already use in ACRN (and which is based on `systemd`) rather than modifying Sysinit V scripts which are more of a legacy thing.

Reply:
@gvancuts  I'll check that. I think using systemd or sysinit V scripts makes no different.